app/jobs/threat_metrix_js_verification_job.rb

```diff
@@ -0,0 +1,88 @@
+class ThreatMetrixJsVerificationJob < ApplicationJob
+  include JobHelpers::StaleJobHelper
+
+  queue_as :default
+
+  discard_on JobHelpers::StaleJobHelper::StaleJobError
+
+  def perform(session_id:)
+    raise_stale_job! if stale_job?(enqueued_at)
```

Comment:
the stale job stuff is mostly for ditching work for expensive user-facing jobs that we don't want to waste time after we've shown the user an error already... I don't think we necessarily need it here